packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 - BugFix: Resolved a bug where extenders adding `ssh` or `base` to `allTypes` on `ProfileCache` will result in duplicate nodes when adding a profile of that type in Zowe Explorer. [#3625](https://github.com/zowe/zowe-explorer-vscode/pull/3625)
 - Fixed error message shown when creating a config file that already exists. [#3647](https://github.com/zowe/zowe-explorer-vscode/issues/3647)
+- Fixed an issue where secure credentials were coerced to non-string types. [#3728](https://github.com/zowe/zowe-explorer-vscode/issues/3728)
```

Comment:
What do we mean by "coerced"? and "types" of what?

Reply:
Resolved in 9da8f96